packages/web/src/utils.test.ts

```diff
@@ -0,0 +1,62 @@
+import { parseProperties } from './utils';
+
+describe('utils', () => {
+  describe('parse properties', () => {
+    describe('strip', () => {
+      it('should allow all properties', () => {
+        const properties = {
+          number: 10,
+          string: 'some-string',
+          boolean: true,
+          nullable: null,
+        };
+
+        const parsed = parseProperties(properties, { strip: true });
+
+        expect(properties).toEqual(parsed);
+      });
+
+      it('should dismiss array and object', () => {
+        const properties = {
+          string: 'some-string',
+          array: [],
+          object: {},
+        };
+
+        const parsed = parseProperties(properties, { strip: true });
+
+        expect({
+          string: 'some-string',
+        }).toEqual(parsed);
+      });
+    });
+
+    describe('throw error', () => {
+      it('should allow all properties', () => {
+        const properties = {
+          number: 10,
+          string: 'some-string',
+          boolean: true,
+          nullable: null,
+        };
+        const parsed = parseProperties(properties, {
```

Comment:
```suggestion
        };
        
        const parsed = parseProperties(properties, {
```

Reply:
if we bug about those new lines, we should just add eslint/prettier rules for that.